selftests/unit/test_utils_diff_validator.py

```diff
@@ -1,25 +1,31 @@
 #!/usr/bin/env python
 
 import os
+import shutil
+import tempfile
 import unittest
 
 from avocado.utils import diff_validator
 
+from .. import temp_dir_prefix
+
 
 class ChangeValidationTest(unittest.TestCase):
 
     def setUp(self):
+        prefix = temp_dir_prefix(__name__, self, 'setUp')
+        self.tmpdir = tempfile.mkdtemp(prefix=prefix)
```

Comment:
What about to use `TemporaryDirectory()` here? With that, there is no usage for `shutil` and it can be used `cleanup()`.

Reply:
Good point! that's actually something that can be done throughout Avocado.